Uno.Gallery/Uno.Gallery.Shared/Views/Styles/NeumorphicStyles.xaml

```diff
@@ -62,142 +65,145 @@
+				<utu:Shadow OffsetX="-4" OffsetY="-4" BlurRadius="8" Spread="0" Color="#615B71" Opacity="0.6" IsInner="True" />
+				<utu:Shadow OffsetX="4" OffsetY="4" BlurRadius="8" Spread="0" Color="#000000" Opacity="0.4" IsInner="True" />
 			</utu:ShadowCollection>
 
 			<!-- ToggleButton -->
 			<utu:ShadowCollection x:Key="ToggleButtonShadows">
-				<utu:Shadow BlurRadius="8" IsInner="False" OffsetX="-2" OffsetY="-2" Opacity="0.8" Spread="0" Color="{StaticResource LightShadowColor4}" />
-				<utu:Shadow BlurRadius="8" IsInner="False" OffsetX="2" OffsetY="2" Opacity="0.8" Spread="0" Color="{StaticResource ShadeShadowColor4}" />
+				<utu:Shadow OffsetX="2" OffsetY="2" BlurRadius="8" Spread="0" Color="#000000" Opacity="0.8" IsInner="False" />
+				<utu:Shadow OffsetX="-2" OffsetY="-2" BlurRadius="8" Spread="0" Color="#79718E" Opacity="0.8" IsInner="False" />
 			</utu:ShadowCollection>
 
 			<!-- IconButton -->
 			<utu:ShadowCollection x:Key="IconButtonShadows">
-				<utu:Shadow BlurRadius="8" IsInner="False" OffsetX="-2" OffsetY="-2" Opacity="0.8" Spread="0" Color="{StaticResource LightShadowColor4}" />
-				<utu:Shadow BlurRadius="8" IsInner="False" OffsetX="2" OffsetY="2" Opacity="0.8" Spread="0" Color="{StaticResource ShadeShadowColor4}" />
+				<utu:Shadow OffsetX="2" OffsetY="2" BlurRadius="8" Spread="0" Color="#000000" Opacity="0.8" IsInner="False" />
+				<utu:Shadow OffsetX="-2" OffsetY="-2" BlurRadius="8" Spread="0" Color="#79718E" Opacity="0.8" IsInner="False" />
 			</utu:ShadowCollection>
 
 			<!-- CheckBox -->
+			<!-- note: 1st shadow's opacity in Hover,Focused,Pressed is calculated from (attached element opacity: [0.08 or 0.12])*(shadow color alpha: 1.0) -->
 			<utu:ShadowCollection x:Key="NeumorphicCheckBoxShadows">
-				<utu:Shadow OffsetX="-1" OffsetY="-1" BlurRadius="4" Opacity="1" Spread="0" Color="{StaticResource LightShadowColor2}" IsInner="False" />
-				<utu:Shadow OffsetX="1" OffsetY="1" BlurRadius="4" Opacity="0.6" Spread="0" Color="{StaticResource ShadeShadowColor2}" IsInner="False" />
+				<utu:Shadow OffsetX="0" OffsetY="0" BlurRadius="8" Spread="8" Color="Transparent" Opacity="0" IsInner="False">
+					<!-- workaround: invisible shadow to prevent major layout difference between states -->
+				</utu:Shadow>
+				<utu:Shadow OffsetX="1" OffsetY="1" BlurRadius="4" Spread="0" Color="#000000" Opacity="0.6" IsInner="False" />
+				<utu:Shadow OffsetX="-1" OffsetY="-1" BlurRadius="4" Spread="0" Color="#615B71" Opacity="1" IsInner="False" />
+			</utu:ShadowCollection>
+			<utu:ShadowCollection x:Key="NeumorphicCheckBoxFocusedShadows">
+				<utu:Shadow OffsetX="0" OffsetY="0" BlurRadius="8" Spread="8" Color="#C7BFFE" Opacity="0.12" IsInner="False" />
+				<utu:Shadow OffsetX="1" OffsetY="1" BlurRadius="4" Spread="0" Color="#000000" Opacity="0.6" IsInner="False" />
+				<utu:Shadow OffsetX="-1" OffsetY="-1" BlurRadius="4" Spread="0" Color="#615B71" Opacity="1" IsInner="False" />
```

Comment:
Just curious why we aren't using StaticResources anymore for the Colors on the Shadows? Not a blocker or anything, something we can refactor later if we want, but wondering if maybe there was an issue using StaticResource?

Reply:
there is no reason to use StaticResource if we are just referencing `/(Light|Dark)ShadowColor\d+/`...
it is unintuitive at best
plus, ludwig has confirmed with me that, the colors themselves dont have a set system right now, so this is fine
and, this will make validation/update easier, and prevent problem like A+B uses shadow1 and later changes for A inadvertently impacting B

Reply:
@kazo0 I agree with @Xiaoy312 that it is easier at the moment for validation/update (as the colors are different between light and dark for the same ShadowCollection).
We will be able to change this if/when we have a set system for this with Ludwig